### DISCUSSION TITLE
Getting help faster

### DISCUSSION THREAD

**Discussion by h-enk** [thuliteio/doks#452] [Q&A]
The faster way to get help? Make sure to provide:

1. __Steps to reproduce__. Clear steps describing how to reproduce the issue. Please please please link to a project repository  if possible, this makes your issue _much_ easier to diagnose (seriously).
2. __Expected result__. What should happen?
3. __Actual result__. What happened.
4. __Environment__. Paste the information here as shown by `npm run check`,

Thanks!

**Comment by cnichte** [comment]
Hello, how can I exchange the default-image.png for a featured-image to each blog article? featured_image: "..." does not work. I always see only the default white image.

**Reply by h-enk** [reply]
When you specify `images` in the frontmatter of a blog post, the first image specified is shown as the featured image.

```md
images: ["bozhin-karaivanov-zs6GW81O538-unsplash.jpg", "kris-mikael-krister-aGihPIbrtVE-unsplash.jpg"]
```

**Reply by cnichte** [reply]
Aaaa, thanks, that does the trick. If you specify it like so:

```
images: ["bozhin-karaivanov-zs6GW81O538-unsplash.jpg"]
```
... it has to be in the Webpackage like so:

```
/blog/my-blogpost/index.md
/blog/my-blogpost/bozhin-karaivanov-zs6GW81O538-unsplash.jpg
```

or so...

```
/blog/my-blogpost/images/bozhin-karaivanov-zs6GW81O538-unsplash.jpg
```

One must not specify the subdirectory!

```
images: ["/images/bozhin-karaivanov-zs6GW81O538-unsplash.jpg"]
```

so, he doesn't find the image... That had me confused.

**Comment by cnichte** [comment]
Is there already news about version 1.0 of Doks?  I am waiting very impatiently for it :-)

**Reply by h-enk** [reply]
I understand that — I'm impatient myself 😄 Can't give you an exact date though. What's crucial is new documentation — including migration guides. Working hard on that now.
